auditbeat/module/auditd/config_test.go

```diff
@@ -28,13 +28,14 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	conf "github.com/elastic/elastic-agent-libs/config"
 	"github.com/elastic/elastic-agent-libs/logp"
 )
 
 func TestConfig(t *testing.T) {
-	logp.TestingSetup()
+	require.NoError(t, logp.TestingSetup())
```

Comment:
This should not be necessary, we already specify in the config that this function is to be ignored.

https://github.com/elastic/beats/blob/28152d53e32b6e0b44c2fd20a5439b47dce4adec/.golangci.yml#L82

Reply:
it's not using the exclude rule for some reason

```
golangci-lint run | rg TestingSetup     

libbeat/common/event_test.go:33:19: Error return value of `logp.TestingSetup` is not checked (errcheck)
	logp.TestingSetup()
libbeat/common/event_test.go:144:19: Error return value of `logp.TestingSetup` is not checked (errcheck)
	logp.TestingSetup()
libbeat/common/event_test.go:202:19: Error return value of `logp.TestingSetup` is not checked (errcheck)
	logp.TestingSetup()
```

I'll investigate why

Reply:
the CI seems to not be picking up the exclude rules even though they work locally; I'll investigate why

Reply:
it seems all the other instances of this function were not being printed out:
```
INFO [runner/max_same_issues] 99/102 issues with text "Error return value of `logp.TestingSetup` is not checked" were hidden, use --max-same-issues
```
removing the parenthesis fixed it